### ISSUE TITLE
Global Usings for Console Apps

### ISSUE THREAD

**Issue by yaserbalaghi** [dotnet/sdk#32130]
Hi everyone,
I suggest that when creating a console project, the following global direction should also be added to the GlobalUsings.cs auto-generation file.

global using static global::System.Console;

So, along with the "Top-Level Statements" feature to teach C# to beginners, the initial confusion will be less and actually it will also be better for professionals.

![Screenshot 2023-04-29 104536](https://user-images.githubusercontent.com/7289120/235289369-fc449aa1-1ff4-4600-934c-14e855dc1faa.png)
![Screenshot 2023-04-29 104601](https://user-images.githubusercontent.com/7289120/235289372-f3866a28-71ab-4cde-8ca3-0fa99825f820.png)

Thank you for your attention.

**Comment by nagilson** [comment]
I see where you're coming from, though I'm not sure we would do that as it'd likely break a lot of people. 
@marcpopMSFT Are we the right place for this feedback? IDK if this should go to Roslyn or if it goes to C# lang

**Comment by marcpopMSFT** [comment]
@nagilson this feels more like a @baronfel question and a template question.

**Comment by baronfel** [comment]
IIRC @DamianEdwards and @KathleenDollard had some input into what set of Global Usings were originally decided on. What heuristics were you all going off of? I'm inclined to thing that `Console` is a step too far, but some kind of data-backed frequency analysis here might be useful.

> the initial confusion will be less and actually it will also be better for professionals

I'm not sure I agree with this assertion on its face. Console is a highly domain-specific area of code, and in most apps I'd say that it is silo'd off into a specific area of the app (notably the CLI arg parsing section) and the rest of your app doesn't ever care about it. To me that's an argument for a file-local `using static`, not a project-global `using static.

**Comment by DamianEdwards** [comment]
Agreed. Adding global static usings is definitely a step beyond what we've thought to date is appropriate for our defaults.
